app/src/main/java/edu/wisc/ece/pinpoint/fragments/ProfilePageFragment.java

```diff
@@ -50,19 +47,22 @@ public void onTabSelected(TabLayout.Tab tab) {
 
             @Override
             public void onTabUnselected(TabLayout.Tab tab) {
-                // Mandatory override meethod left intentionally blank because we will not implement onTabUnselected
+                // Mandatory override method left intentionally blank because we will not
+                // implement onTabUnselected
             }
 
             @Override
             public void onTabReselected(TabLayout.Tab tab) {
-                // Mandatory override meethod left intentionally blank because we will not implement onTabReselected
+                // Mandatory override method left intentionally blank because we will not
+                // implement onTabReselected
```

Comment:
Is this split across 2 lines because of a line length rule?

Reply:
Yeah the line length rule. I'll disable it for comments to keep it on one line.